package.json

```
@@ -26,6 +26,7 @@
     "test": "swagger-cli validate --no-schema web_deploy/swagger.yaml",
     "start": "gulp serve",
     "deploy-branch": "node ./scripts/deploy-branch.js",
-    "sync-generated-specs": "node ./scripts/sync-generated-specs.js"
+    "sync-generated-specs-new": "node ./scripts/sync-generated-specs-new-env.js",
+    "sync-generated-specs-old": "node ./scripts/sync-generated-specs-old-env.js"
```

Maybe we can use more future proofed terms throughout (for when we one day remove legacy :) ):
```suggestion
    "sync-generated-specs": "node ./scripts/sync-generated-specs.js",
    "sync-generated-specs-legacy-aws-env": "node ./scripts/sync-generated-specs-legacy-aws-env.js"
```